File: aDatos/jdbcTest/examenAlejandroMarino/domain/src/main/java/domain/modelo/Equipo.java
package domain.modelo;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Equipo {
    private String nombre;
    private String entrenador;
    private List<Jugador> jugadores;
}

File: ExamenAlejandroMarino2/app/src/main/java/com/example/examenalejandromarino/domain/modelo/Equipo.kt
package com.example.examenalejandromarino.domain.modelo

data class Equipo(
    val id: Int = 0,
    val nombre:String = "",
    val nacionalidad: String = "",
    val puesto: Int = 0,
    val componentes: List<Componente>? = emptyList(),
)